### ISSUE TITLE
Paper Request: "FocalPO: Enhancing Preference Optimizing by Focusing on Correct Preference Rankings" ACL main 2025

### ISSUE THREAD

**Issue by TongLiu-github** [liushunyu/awesome-direct-preference-optimization#2]
Dear authors,  

Thanks for this survey paper. We are glad to share our recent ACL main paper. 

Motivation is quite strightforward: Chen et al., 2024 empirically finds that DPO training rarely improves these misranked preference pairs, despite its gradient emphasizing on these cases. We accordingly add a simple factor to DPO loss to make DPO **focus on "more correct" samples**. **With the introduced hyperparameter fixed**, it consistently outperforms DPO on Arena-hard and Alpaca Eval for both models. 

This is the **first** paper to modify DPO from the perspective of **positive/negative** samples. After this paper, recently several works exploring the influence of positive/negative samples in GRPO also came out. 

Thanks for considering possible citing.

**Comment by liushunyu** [comment]
Thanks for sharing such nice work! We have updated the repository. We will also added this work to the next version of the paper :)
